assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -262,6 +264,7 @@ const baseActions = {
 			SettingsEditComponent,
 			SettingsViewComponent,
 			SetupComponent,
+			SettingsSetupIncompleteComponent,
```

Comment:
This is missing a definition in `moduleDefaults` above:

https://github.com/google/site-kit-wp/blob/8c37f05d1f1a951f91292de6ec52209a3a82a956/assets/js/googlesitekit/modules/datastore/modules.js#L67-L68

This can be set to the `DefaultSettingsSetupIncomplete` component type as long as we pass the `slug` unconditionally in `SettingsRenderer`.